configs/train_configs/sft/olmo_7b_17_remix_sft.yaml

```diff
@@ -0,0 +1,27 @@
+# Note, the exact model was trained on TPUs in a different repo
+# Example file for mixing, use exact tulu 2 mixture for replication allenai/tulu-v2-sft-mixture
+model_name_or_path: allenai/OLMo-1.7-7B-hf
+model_revision: main
+use_flash_attn: true
+tokenizer_name: allenai/OLMo-1.7-7B-hf
+use_slow_tokenizer: true
+dataset_mixer:
+  allenai/wildjailbreak: 262000
+  /net/nfs.cirrascale/mosaic/oe-safety-datasets/vanilla_harmful_v.03/sampled_data/gpt-3.5-turbo_synthetic_harm_v03.jsonl: 21876
+  /net/nfs.cirrascale/mosaic/oe-safety-datasets/wildchat_lmsys_sexual/gpt4_lmsys_wildchat_dedup_50ksampled.jsonl: 16888
+  allenai/tulu-v2-sft-mixture: 326154
```

Comment:
Maybe it's helpful to specify the percentage instead of the absolute value?

Reply:
Percentage also works!